### ISSUE TITLE
Default value on settings is not set on new installations

### ISSUE THREAD

**Issue by mitogh** [WordPress/performance#247]
## Bug Description

When the plugin is installed in a new installation, the default settings for the modules are not used but the UI on the settings reflect the usage of the default settings for each module, which would indicate visualize that the settings are `ON` but the modules are not in use.

## Steps to reproduce

1. Create a new WP installation or remove all the options of your current site
2. Install the latest version of the plugin
3. Go to Performance > Settings and observe the UI that shows the active modules. Make sure to not click the **Save** button.
4. Go to the media library and upload a new image
5. Insert the image into the content and observe no WebP images were created.

## Screenshots

![2022-03-21_11-49](https://user-images.githubusercontent.com/3921289/159333626-39b6aff5-4caa-4686-9716-0bbfa901883c.png)
![2022-03-21_11-49_1](https://user-images.githubusercontent.com/3921289/159333623-4ce90655-9d8f-4d1b-a152-9ad14c7d422d.png)

Markup of the screenshot:

```html
<img src="https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-1024x678.jpg" alt="" class="wp-image-7" srcset="https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-1024x678.jpg 1024w, https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-300x199.jpg 300w, https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-768x509.jpg 768w, https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-1536x1017.jpg 1536w, https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-2048x1356.jpg 2048w, https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-1200x795.jpg 1200w, https://sadfire.s2-tastewp.com/wp-content/uploads/2022/03/candra-winata-CYOFvtpOIpU-unsplash-1980x1311.jpg 1980w" sizes="(max-width: 1024px) 100vw, 1024px" width="1024" height="678">
```

## Workaround

 Currently, the workaround for this bug is

- once, the plugin is activated
- go to `Performance > Settings` 
- click on **Save**.

**Comment by felixarntz** [comment]
The problem here is that the modules setting is only registered (via `register_setting()`) on the `init` action, but the active modules are loaded as soon as the plugin is loaded which means the default value isn't hooked into the necessary filter at this time.

We can fix this by passing the same default manually in our `perflab_get_module_settings()` function. Registering settings is expected to happen on `init`, so we shouldn't just run that immediately. But modifying the above function so that the default is _always_ taken into account, regardless of timing, makes sense.

**Comment by felixarntz** [comment]
Fixed by #248 - which just got merged, so the "hot fix" `1.0.0-beta.3` can go out today.
